src/Functions/plotIntensityProfileAndPeaks.m

```diff
@@ -1,13 +1,27 @@
-%% plotIntensityProfileAndPeaks
+function [peakLocs, peakWaveCenters, peakValues, threshIntensityProfile, threshold] = getIntensityPeaks(intensityProfile, skelScaleMm, filterIdxs);
 %
-% Andreas Husch
-% Centre Hospitalier de Luxembourg, Dep. of Neurosurgery /
-% University of Luxembourg - Luxembourg Centre for Systems Biomedicne
-% 2017
-% mail@andreashusch.de, husch.andreas@chl.lu
-function plotIntensityProfileAndPeaks(intensityProfile, skelScaleMm)
+% USAGE:
+%
+%    [peakLocs, peakWaveCenters, peakValues, threshIntensityProfile, threshold]
+%    = getIntensityPeaks(intensityProfile, skelScaleMm, filterIdxs);
+%
+% INPUTS: 
+%    intensityProfile:      
```

Comment:
@adhusch Could you please provide a description for the inputs and outputs?

Reply:
@adhusch ?

Reply:
intensityProfile a 1d intensity profile
skelScaleMm estimated scale (mm per intensity sample)